.github/workflows/types-integration.yml

```diff
@@ -146,3 +146,24 @@ jobs:
       - name: Run TSC
         working-directory: eslint-js
         run: npm run build:types --workspace eslint-visitor-keys
+
+  are-the-types-wrong:
+    name: Are the types wrong?
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout eslint
+        uses: actions/checkout@v4
+        with:
+          path: eslint
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "lts/*"
+
+      - name: Install Packages
+        working-directory: eslint
+        run: npm install
+
+      - name: Check validity of type definitions
+        working-directory: eslint
+        run: npx @arethetypeswrong/cli@latest -P
```

Comment:
I think this command could come in handy also during development. Would it be possible to include it as an npm `"script"` in package.json that one could run manually, and have that script called in this workflow with `npm run`?

Reply:
Of course.